src/tensorwaves/model.py

```diff
@@ -109,13 +119,6 @@ def __init__(
         if not all(map(lambda p: isinstance(p, sp.Symbol), parameters)):
             raise TypeError(f"Not all parameters are of type {sp.Symbol}")
 
-        if not set(parameters) <= set(expression.free_symbols):
```

Comment:
why is this removed?

Reply:
Because if you take a sub-expression that does not feature all the parameters that were in the original model, this will crash. I don't think a check like this is necessary here.

Reply:
Possible, but then the caller always have to filter those beforehand useing `free_symbols`.

Reply:
Added the chack back but as warning e08f084